packages/extension-ui/src/Popup/Accounts/index.tsx

```diff
@@ -43,7 +43,7 @@ function Accounts ({ className }: Props): React.ReactElement {
 
   return (
     <>
-      {(hierarchy.length === 0)
+      {(hierarchy?.length === 0)
```

Comment:
```suggestion
      {!hierarchy?.length
```

The original doesn't quite do the trick. since `undefined !== 0`

Actually, what this most-probably needs to do is something like -

```js
if (! hierarchy) {
  return null;
}
```

Since we most probably don't want anything to show when we don't know, i.e. things are not in-place.